cloud/scope/powervs_cluster.go

```diff
@@ -1136,9 +1139,7 @@ func (s *PowerVSClusterScope) ReconcileVPCSubnets() (bool, error) {
 		}
 		if vpcSubnetID != "" {
 			s.V(3).Info("Found VPC subnet in IBM Cloud", "id", vpcSubnetID)
-			s.SetVPCSubnetID(*subnet.Name, infrav1beta2.ResourceReference{ID: &vpcSubnetID, ControllerCreated: ptr.To(false)})
-			// check for next subnet
-			continue
+			s.SetVPCSubnetStatus(*subnet.Name, infrav1beta2.ResourceReference{ID: &vpcSubnetID, ControllerCreated: ptr.To(false)})
```

Comment:
why continue removed from here? We are supposed to move to next subnet?

Reply:
Removed by mistake. Rectified!